README.md

```diff
@@ -74,11 +75,14 @@ To deploy a production release:
             2. create a new commit with the CHANGES.md message
             3. create a tag `v[new-version-string]` with the CHANGES.md message
 4. Push current branch and tag to GitHub
+  - `git push origin master`
+  - `git push origin v<new version>`
 5. Create a GitHub Release
     1. Find the new tag at https://github.com/concord-consortium/activity-player/tags open it, and edit it
     2. Copy the title from CHANGES.md
     3. Copy the content from CHANGES.md
-6. QA the built version at `https://activity-player.concord.org/versions/v[new-version-string]``
+    4. Hit "Publish Release" button
+6. QA the built version at `https://activity-player.concord.org/version/v[new-version-string]`
```

Comment:
This URL works with a slash at the end : `https://activity-player.concord.org/version/v[new-version-string]/` - doesn't work without it.